scratch-vm/rspack.config.js

```
@@ -0,0 +1,215 @@
+        // it must be set explicitly here.
+        externalsType: 'commonjs2',
+        externals: nodeExternals
+    }),
+    // Server
+    defaultsDeep({}, base, {
+        target: 'node',
+        entry: {
+            cli: './src/server/cli.js'
+        },
+        output: {
+            library: undefined,
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)